index.html

```diff
@@ -64,108 +64,28 @@ <h2 class="headline-section__sub-title">
-					</p>
-					<ul class="tech">
-						<li class="tech__type">html</li>
-						<li class="tech__type">css</li>
-						<li class="tech__type">javaScript</li>
-					</ul>
-					<button type="button" class="btn see-project">See Project</button>
-				</div>
-			</div>
-			<div class="work-section__card">
-				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(1).png" alt="card snap"></div>
-				<div class="work-section__card-img desktop order--two"><img src="./images/Card-images/SnapshootPortfolioDesktop2.png" alt="card snap"></div>
-				<div class="work-section__card--info">
-					<h3 class="project-name">Multi-post Stories</h3>
-					<div class="range">
-						<p class="range__canopy mobile">CANOPY</p>
-						<p class="range__canopy desktop">FACEBOOK</p>
-						<div class="range__circle"></div>
-						<p class="range__txt mobile">Back End Dev</p>
-						<p class="range__txt desktop">Fullstack Dev</p>
-						<div class="range__circle"></div>
-						<p class="range__txt">2015</p>
-					</div>
-					<p class="general__desc">
-						Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
-					</p>
-					<ul class="tech">
-						<li class="tech__type">html</li>
-						<li class="tech__type">css</li>
-						<li class="tech__type">javaScript</li>
-					</ul>
-					<button type="button" class="btn see-project">See Project</button>
-				</div>
 			</div>
-			<div class="work-section__card">
-				<div class="work-section__card--img mobile"><img src="./images/SnapshootPortfolio(2).png" alt="card snap"></div>
- 				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop3.png" alt="card snap"></div>
-				<div class="work-section__card--info">
-					<h3 class="project-name mobile">Tonic</h3>
-					<h3 class="project-name desktop">Facebook 360</h3>
-					<div class="range">
-						<p class="range__canopy mobile">CANOPY</p>
-						<p class="range__canopy desktop">FACEBOOK</p>
-						<div class="range__circle"></div>
-						<p class="range__txt mobile">Back End Dev</p>
-						<p class="range__txt desktop">Full-stack Dev</p>
-						<div class="range__circle"></div>
-						<p class="range__txt">2015</p>
-					</div>
-					<p class="general__desc">
-						Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
-					</p>
-					<ul class="tech">
-						<li class="tech__type">html</li>
-						<li class="tech__type">css</li>
-						<li class="tech__type">javaScript</li>
-					</ul>
-					<button type="button" class="btn see-project">See Project</button>
-				</div>
+			<div class="modals">
+				<div class="pop-up--container active">
+			<div class="pop-up__header">
+				<h3 class="project-name">.</h3>
+				<div class="close-btn"><img src="./images/x.png" alt="close"></div>
```

Comment:
## [OPTIONAL]
- [ ] I would suggest positioning very well this `close-btn` and also putting a pointer cursor when this is being hovered.